PhpcsChanged/CacheManager.php

```diff
@@ -127,7 +127,9 @@ public function setCacheVersion(string $cacheVersion): void {
 		$this->cacheObject->cacheVersion = $cacheVersion;
 	}
 
-	public function getCacheForFile(string $filePath, string $type, string $hash, string $phpcsStandard): ?string {
+	public function getCacheForFile(string $filePath, string $type, string $hash, string $phpcsStandard, string $warningSeverity, string $errorSeverity): ?string {
+		$phpcsStandard = $this->getPhpcsStandardCacheKey( $phpcsStandard, $warningSeverity, $errorSeverity );
```

Comment:
Can we briefly mention this overloading of the standard up on line 17 where the cache keys are documented? That will probably help in a few years when we are like "why does this have a standard of VIP57??" 😉